frontend/src/components/PresetList.tsx

```diff
@@ -0,0 +1,76 @@
+"use client"
+
+import React, { ReactNode } from "react"
+
+import Link from "next/link"
+
+import classNames from "classnames"
+
+import AsyncButton from "@/components/AsyncButton"
+import Button from "@/components/Button"
+import LoadingSpinner from "@/components/loading.svg"
+import { PlatformIcon } from "@/components/PlatformSelect/PlatformIcon"
+import styles from "@/components/ScratchList.module.scss"
```

Comment:
Aah, let's not import styles from another component like this.

Much better would be to port the styles into tailwind, and/or figure out how to abstract these into components so they can be reused properly.

The goal long-term is to stop using SCSS modules in favour of Tailwind and components.

Reply:
If you like this can be a TODO if you make an issue

Reply:
I change to tailwindcss classes ✅